IFC4x3/PropertyConstants/o/OTHER_1e965qwjTCJP79vQ7RW1XB.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="OTHER_1e965qwjTCJP79vQ7RW1XB" Name="OTHER" UniqueId="68246174-ead7-4c4d-91c9-e5a1db80184b">
+	<Documentation>Another, user defined, type of borehole.</Documentation>
```

Comment:
Why not name it USERDEFINED then?
```suggestion
	<Documentation>User defined, type of borehole.</Documentation>
```

Reply:
Please change the description to 'User defined, other type of borehole'
Please leave the constant 'OTHER' as is